.ci/.jenkins-loggers.yml

```
@@ -0,0 +1,16 @@
+---
+loggers:
+  - REPO: "ecs-dotnet"
+    SPEC_PATH: "tests/Elastic.CommonSchema.Tests/Specs"
+  - REPO: "ecs-logging-go-logrus"
+    SPEC_PATH: "internal/spec"
+  - REPO: "ecs-logging-go-zap"
+    SPEC_PATH: "internal/spec"
+  - REPO: "ecs-logging-java"
+    SPEC_PATH: "ecs-logging-core/src/test/resources/spec"
+  - REPO: "ecs-logging-js"
```

Sorry for the noise, in https://github.com/elastic/ecs-logging-nodejs/issues/48 I've just changed this repo to be named "ecs-logging-nodejs". :)

```suggestion
  - REPO: "ecs-logging-nodejs"
```